doc/manual/manual.tex

```diff
@@ -10036,6 +10036,49 @@ \subsection{Extending the basic solver}
 \cite{BHK07,BK99m}.}
 
 
+\subsection{Testing \aspect}
+\label{sec:testing}
+
+\aspect{} makes use of a large suite of tests to ensure correct behavior.
```

Comment:
Maybe add a sentence that the tests are run automatically with each change to the github repository, but that it is good practice to add new ones, when you add new features.